### ISSUE TITLE
[11.02] upgrade do not track the activity timing

### ISSUE THREAD

**Issue by D4708** [wakatime/atom-wakatime#106]
Dashboard not tracking the coding activity by following message in the tray:

`downloading/extracting wakatime-cli...`

![Screenshot from 2021-05-20 16-49-29](https://user-images.githubusercontent.com/76843085/118990220-5bff9000-b98b-11eb-89c5-f9ac225b7f3a.png)

Atom 1.57.0 / Linux
Plugin version: 11.02

**Comment by alanhamlett** [comment]
What's the message in your Atom Developer Console window? (`View → Developer → Toggle Developer Tools`)

Also check your `~/.wakatime.log` file for error messages.

**Comment by D4708** [comment]
Ctrl+Shift+i

`Uncaught (in promise) Error: ENOENT: no such file or directory, chmod '/home/***/.wakatime/wakatime-cli-linux-amd64'`

that directory does not exist

`~/.wakatime.log`

is empty

**Comment by alanhamlett** [comment]
Fixed in v11.0.3.

**Comment by D4708** [comment]
thank you!

**Comment by hayesall** [comment]
Really similar error with `extracting wakatime-cli...` in `v11.0.4`:

(Atom 1.49.0 / Linux)

```console
[WakaTime] [DEBUG] [wakatime.js:50] Initializing WakaTime v11.0.4...

[WakaTime] [DEBUG] [wakatime.js:330] GitHub API Response 304

[WakaTime] [DEBUG] [wakatime.js:402] Downloading wakatime-cli from https://github.com/wakatime/wakatime-cli/releases/download/v1.7.4/wakatime-cli-linux-amd64.zip

[WakaTime] [DEBUG] [wakatime.js:420] Extracting wakatime-cli.zip file...

Uncaught (in promise) Error: ENOENT: no such file or directory, chmod '/home/hayesall/.wakatime/wakatime-cli-linux-amd64'
    at Object.chmodSync (fs.js:1079)
    at wakatime.js? [sm]:328
    at Generator.next (<anonymous>)
    at step (wakatime.js? [sm]:370)
```

I turned on logging in settings, but `.wakatime.log` only has old cases of "could not detect the language of file."

**Comment by alanhamlett** [comment]
@hayesall first try restarting your Atom editor. Next, if that doesn't work try downloading [wakatime-cli](https://github.com/wakatime/wakatime-cli/releases/download/v1.7.4/wakatime-cli-linux-amd64.zip) and extracting into `/home/hayesall/.wakatime/` then restarting Atom.

**Comment by alanhamlett** [comment]
Looks like there was a bug in our release where it stayed as Draft. I've published it now, so it should download v1.7.4 when you relaunch Atom?

**Comment by hayesall** [comment]
Fixed! Thank you. I'll close the issue I migrated.
